src/mito2/src/region.rs

```diff
@@ -163,6 +165,23 @@ impl MitoRegion {
     fn estimated_wal_usage(&self, memtable_usage: u64) -> u64 {
         ((memtable_usage as f32) * ESTIMATED_WAL_FACTOR) as u64
     }
+
+    pub async fn apply_edit(
```

Comment:
It's not recommended to make this method public. We hide the `MitoRegion` behind the engine and only the worker should be able to call `apply_edit()`. Otherwise, there might be some concurrent issues.